include/odp/api/spec/ipsec.h

```diff
@@ -772,6 +773,71 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/**
+ * IPSEC SA information
+ */
+typedef struct odp_ipsec_sa_info_t {
+	/** Copy of IPSEC Security Association (SA) parameters */
+	odp_ipsec_sa_param_t param;
+
+	/** IPSEC SA direction dependent internal parameters */
+	union {
+		/** Inbound specific internal parameters */
+		struct {
+			/** Additional SA lookup parameters. */
+			struct {
+				/** IP destination address (NETWORK ENDIAN) to
+				 *  be matched in addition to SPI value. */
+				uint8_t dst_addr[ODP_IPV6_ADDR_SIZE];
+			} lookup_param;
```

Comment:
How does the caller know if the address is IPv4 or IPv6? In the corresponding config API there is also IP version field.

Reply:
Didn't include the IP version separately as it is already available in the copy of SA param(param.inbound.lookup_param.ip_version).